src/Magnum/DebugTools/ColorMap.h

```diff
@@ -204,6 +204,20 @@ MAGNUM_DEBUGTOOLS_EXPORT Containers::StaticArrayView<256, const Vector3ub> infer
 */
 MAGNUM_DEBUGTOOLS_EXPORT Containers::StaticArrayView<256, const Vector3ub> viridis();
 
+/**
+@brief Cool-Warm colormap
```

Comment:
This is the "Smooth" version, right? I'm wondering if it would make sense to add the other one ... or maybe just rename this to Smooth Cool-Warm to avoid confusion and reserve a potential place for the Bent one in the future.

Reply:
Yes its the smooth version. I'll rename it. I feel like the bent version looks looks pretty similar to the smooth version. So until somebody actually wants to use it, I would suggest to leave it out?

Reply:
I can see a use case where you compare two color-mapped images and the small difference between the smooth and bent variant would significantly alter the result. So better be clear :)

Reply:
Agree. I added the bent version as well :)